### DISCUSSION TITLE
openPageForResult可以用在fragment之间跳转传值吗

### DISCUSSION THREAD

**Discussion by chenyao1208** [xuexiangjys/XUI#141] [Q&A]
openPageForResult可以用在fragment之间跳转传值吗，我试了报错，如果不可以使用openPageForResult，那我应该使用哪种方式，谢谢。
![image](https://user-images.githubusercontent.com/21217350/176083338-8ce36d86-d4e9-4c69-817f-83e39f0c5201.png)

**Comment by xuexiangjys** [comment]
可以跳转传值,具体参见官方文档:
https://github.com/xuexiangjys/XPage
